binder/environment-EDITABLE.yml

```diff
@@ -15,3 +15,4 @@ dependencies:
   - mamba
   - pip
   - sphinxcontrib-bibtex
+  - cookiecutter
```

Comment:
This error occurred when I tried to run `./lock-environment.sh` locally:

```PackagesNotFoundError: The following packages are not available from current channels:

  - mamba=0.17
  - conda-lock=0.13

Current channels:

  - https://repo.anaconda.com/pkgs/main/linux-64
  - https://repo.anaconda.com/pkgs/main/noarch
  - https://repo.anaconda.com/pkgs/r/linux-64
  - https://repo.anaconda.com/pkgs/r/noarch

To search for alternate channels that may provide the conda package you're
looking for, navigate to

    https://anaconda.org

and use the search bar at the top of the page.


Could not find conda environment: CondaLock
You can list all discoverable environments with `conda info --envs`.
```

Reply:
I tried running this and did NOT run into an error. However, my conda is set to default to `-c conda-forge`, and all the packages listed as output are from conda-forge. Your current channels don't include conda-forge, so I'm guessing that's might be the problem. @scottyhq, can we add a channel to the `conda` command in `binder/lock-environment.sh`?

Reply:
yep, we can change the first command in the lock-environment.sh script to:
 `conda create -q -y -n CondaLock -c conda-forge conda-lock=0.13 mamba=0.17`